ui/main.qml

```diff
@@ -345,13 +345,12 @@ StatusWindow {
         id: notificationWindow
     }
 
-    MacTrafficLights {
-//        parent: Overlay.overlay
+    MacTrafficLights { // FIXME should be a direct part of StatusAppNavBar
```

Comment:
> // FIXME should be a direct part of StatusAppNavBar

:+1:

It could be done similarly to how we already propagate `profileComponent` to the `StatusAppNavBar`.